index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>CV Dzmitry Kulevich</title>
+	<meta charset="utf-8">
+	<link rel="stylesheet" href="cv.css">
+</head>
+<body>
+	<main>
+		<h1>CV Dzmitry Kulevich</h1><hr>
+		<section id="personal-information">
+			<img src="cv-photo.jpg" alt="Photo for CV" width="200" height="230">
+			<h3>Personal Information</h3>
+			<table>
+				<tbody>
+					<tr>
+						<td><span class=bold>Name:</span></td>
+						<td><span class="italic">Dzmitry</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Last Name:</span></td>
+						<td><span class="italic">Kulevich</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Date of birth:</span></td>
+						<td><span class="italic">20/11/1987</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Nationality:</span></td>
+						<td><span class="italic">Belarus</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Marital status:</span></td>
+						<td><span class="italic">Married</span></td>
+					</tr>
+				</tbody>
+			</table>
+		</section>
+		<section id="contact-information">
+			<h3>Contact Information</h3>
+			<table>
```

Comment:
it's ok to use table here but for semantics I'd advice to put it in ```address``` tag https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address
it can contain not only address but also any other contact information (phone, email, etc.)